README.md

```diff
@@ -168,3 +168,14 @@ EMLinux currently supports following machines.
 - qemu-arm64
 - generic-x86-64
 - raspberrypi3bplus-64
+
+## Sample Recipe
+
+There are some sample recipes under doc/samples/recipes-sample/ directory. Currently you can see following sample recipes.
+
+- openssl-apt
```

Comment:
Please add ":" like `openssl-apt:`

> - openssl-apt This recipe downloads openssl source package using apt://. This recipe shows how to change build option and how to add new patch for source code.

> - openssl-apt: This recipe downloads openssl source package using apt://. This recipe shows how to change build option and how to add new patch for source code.

Reply:
add white space instead of ':'  I think this style is better.

Reply:
Thanks!